syntax = "proto3";

package nml;

import "channel.proto";

message SetChannel {
    repeated Channel channel = 1;
}